generator/sequence_transform_config_sample.json

```diff
@@ -1,10 +1,11 @@
 {
-    "rules_file_name": "./magic.txt",
-    "magic_chars": "☆✵★✪",
-    "wordbreak_char": "⎵",
-    "output_func_chars": "↻⇑",
-    "comment_str": "//",
+    "rules_file_name": "sequence_transform_dict.txt",
+    "sequence_token_symbols": "👆👍★✪",
+    "wordbreak_symbol": "⎵",
+    "output_func_symbols": "↻⇑",
```

Comment:
`output_func_symbols` is really vague, I honestly can't figure a better name for it, but its **very** vague

Reply:
We'll address this later, when we have all the options defined.